src/components/Repo.tsx

```diff
@@ -12,12 +15,32 @@ const Repo: React.FC<RepoProps> = ({ data, key }) => {
       href={"https://github.com" + data.name}
       target="_blank"
       rel="noopener noreferrer"
-      className="bg-slate-700 p-4 rounded-lg shadow-md hover:shadow-lg transition-shadow duration-300"
+      className="repo-card bg-slate-700 p-4 rounded-lg shadow-md hover:shadow-lg transition-shadow duration-300"
       key={key}
     >
-      <p className="text-xl font-semibold text-gray-200">{data.name}</p>
-      <p className="text-gray-400">Stars: {data.stars}</p>
-      <p className="text-gray-400">Forks: {data.forks}</p>
+      <div className="flex items-center">
+        <p className="text-xl font-semibold text-gray-200">{data.name}</p>
+
+        {/* Language Badge - Only show if language is specified */}
+        {data.language ? (
+          <span className="ml-2 inline-block bg-blue-500 text-white text-xs font-medium px-2.5 py-0.5 rounded">
```

Comment:
Use something like [github-colors](https://www.npmjs.com/package/github-colors) for custom bg colors.

Reply:
Done!